documentation/modules/ROOT/pages/configuration/avro.adoc

```diff
@@ -121,7 +121,7 @@ In your environment, you might want to use a provided {prodname} container to de
 
 .Prerequisites
 
-* You have the cluster administrator access to an OpenShift cluster.
```

Comment:
Whoops, good catch. There's also a "serializaion" typo a few lines before you could fix.

Reply:
Done